app/views/hyrax/batch_ingest/batches/summary/_object_summary.html.erb

```diff
@@ -0,0 +1,29 @@
+<section class="batch-ingest-summary-section">
+  <h3><%= total %> Objects Ingested <sup>*</sup></h3>
+  <% unless rows.empty? %>
+    <div class="table-responsive">
+      <table class="table">
+        <thead>
+          <tr>
+            <th style="width: 5%;">#</th>
+            <th style="width: 5%;">%</th>
+            <th>Object Type</th>
+          </tr>
+        </thead>
+        <tbody>
+          <% rows.each do |row| %>
+            <td><%= row[:count] %></td>
+            <td><%= row[:percent] %></td>
+            <td><%= row[:object_type] %></td>
+          <% end %>
+        </tbody>
+      </table>
+    </div>
+  <% end %>
+
+  <div class="footnote">
+    * this shows 1 ingested object per successful batch item, however each batch
+      item may actually create more than 1 object. It will depend on how the ingester
```

Comment:
This message is helpful.  I think mentioning "ingester" would probably be confusing to an end-user.  I wonder if there is different wording that might make this clearer.  Maybe something other than "Object" in the header?  I'd been trying to use "Top-level object" but that feels confusing as well since it all really depends on your implementation.  Maybe "Batch item object" even though that feels redundant?

Reply:
how about?...
```
* This shows the number of objects referred to by successfully completed batch items. However, there may have been more objects created during the ingest process.
```